app/assets/javascripts/list.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
It looks like this isn't used. Can it be removed?

There's a bunch of other unused files here as well (including tests, fixtures, and helpers). Maybe they could be removed to?

Reply:
Yeah, they were generated automatically when I generated the models... I'll remove these and opt for creating the relevant files myself next time 🤔